src/main.h

```diff
@@ -1,9 +1,53 @@
 #include <Arduino.h>
+#include <ArduinoJson.h>
+#include <PubSubClient.h>
 
 #define CLOUD4RPI_SERVER "mq.cloud4rpi.io"
 
+#define RECONNECT_TIMEOUT 3000
+#define DISABLE_RETRY 0
```

Comment:
Currently it works as `RETRY_ONCE`
We'd better make it `NOT_RECONNECT` with the following `ensureConnection` implementation:

```
bool Cloud4RPi::ensureConnection(int maxReconnectAttempts, int reconnectTimeout) {
    if (maxReconnectAttempts == 0)
        return this->connected();
    int attempt = 0;
    bool forever = maxReconnectAttempts <= RETRY_FOREVER;  // Note that I covered all negative numbers
    while (!this->connected() && (endlessReconnection || attemptNumber < reconnectAttempts)) {
        Serial.print("Attempting to connect to Cloud4PRi broker, (#");
```